Harvest.Web/ClientApp/src/Quotes/QuoteContainer.tsx

```diff
@@ -150,10 +151,17 @@ export const QuoteContainer = () => {
       body: JSON.stringify(quote),
     });
 
+    toast.promise(request, {
+      loading: "Saving Quote",
+      success: "Quote saved.",
+      error: "Something went wrong, please try again",
```

Comment:
@srkirkland I tried to throw an exception and return a bad request. Neither displayed the error.